modules/ocf/templates/kubectl/kubectl.conf.erb

```diff
@@ -0,0 +1,20 @@
+users:
+- name: ocfstaffer
+  user:
+    exec:
+      command: "/opt/share/utils/sbin/gen-kubernetes-cert"
```

Comment:
I think it's preferable to not specify the full path, in case the script moves or something. This would also let staffers patch the script and put the patched version in their `PATH` to test changes.

Reply:
staffers can also just change the `KUBECONFIG` variable to test changes. the likelihood of this path changing is pretty low